extension-manifest-v2/app/scss/ghostery_ui.scss

```diff
@@ -1 +1 @@
-@import "@ghostery/ui/css";
+// @import "@ghostery/ui/src/styles/pages.css";
```

Comment:
are those meant to be commented?

Reply:
Leftover, the file should be removed completely.